index.bs

```diff
@@ -779,27 +845,25 @@ Registry instances are initially created with the internal slots described in th
 1. Return _p_.
 </emu-alg>
 
-<h4 id="request-link" aoid="RequestLink">RequestLink(loader, key)</h4>
+<h4 id="request-link" aoid="RequestLink">RequestLink(entry)</h4>
 
 <emu-alg>
-1. Let _entry_ be EnsureRegistered(_loader_, _key_).
 1. Let _linkStageEntry_ be GetStage(_entry_, "link").
 1. If _linkStageEntry_ is *undefined*, return a promise resolved with *undefined*.
-1. Return the result of transforming RequestSatisfy(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
+1. Return the result of transforming RequestSatisfy(_entry_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
```

Comment:
same as above, entry and entry after resolving the promise, while are suppose to be the same...